icechunk/src/session.rs

```diff
@@ -1014,9 +1014,9 @@ async fn updated_nodes<'a>(
         .chain(change_set.new_nodes_iterator(manifest_id)))
 }
 
-async fn get_node<'a>(
+async fn get_node(
```

Comment:
Why do you need all these new lifetimes @mpiannucci ? What breaks if you don't?

Reply:
I got rid of them, rust 1.84 doesnt let clippy pass if you dont